### ISSUE TITLE
Odd relationships showing up in term "neighborhood" view

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#545]
response to biotic stimulus

has some relationship to "cross-streak test evidence"

looks strange?

![amigo bug](https://user-images.githubusercontent.com/7359272/48318383-da7b5a00-e5f7-11e8-93c9-a737af2b8623.jpg)

**Comment by kltm** [comment]
Term:
http://amigo.geneontology.org/amigo/term/GO:0009607

Lineage graph: 
http://amigo.geneontology.org/amigo/term/GO:0009607#display-lineage-tab

"Neighborhood" as above.

**Comment by kltm** [comment]
These additional relations only occur in `neighborhood_graph_json` and `neighborhood_limited_graph_json`, `topology_graph_json` and others are clean.
There was something we had with Planteome that concerned this I think...

**Comment by ValWood** [comment]
I'm pretty sure this would be very confusing to the average AMiGO user.

**Comment by kltm** [comment]
@cmungall Is this actually operating as intended (see https://github.com/geneontology/amigo/issues/220 ), short the failure to have a proper default icon, or do we need/want to have a slightly tighter filter in the owltools function?

@ValWood The neighborhood view is there to give a complete look at the immediate area around a term using *all* of the links available in `go-plus`, including, apparently, some of the more "exotic" ones.

**Comment by ukemi** [comment]
Maybe add the labels for the relations? Then they would make more sense if people didn't want to look up what they are.

**Comment by kltm** [comment]
@ukemi Yes, a little more verbiage and explanation all around if this is the desired/intended behavior. The way the plumbing works though, we'd need to create an internal list of likely relations (or modify the loader to add the label information).

**Comment by ValWood** [comment]
Maybe, but this particular one really doesn't make any sense ;) Why would you put evidence here?

**Comment by ValWood** [comment]
Evidence is for annotation not for ontology terms. 

 or if not,  I  want to add 
"inferred from direct assay"  and "inferred from mutant phenotype" to every term in the 
catalytic activity branch, for example. 

Why add confusing things?

**Comment by pgaudet** [comment]
I agree, it's confusing. How can a user tell where this is coming from ?

**Comment by kltm** [comment]
@ukemi Or possibly a quirk/problem in go-plus, as this is not a general problem?

**Comment by cmungall** [comment]
All relations should have labels. Let's just filter out if we don't have a relation label for some reason. We should also show source of axioms.

In this case the relationship is simultaneously correct (if kind of weird) and of zero use to almost all AmiGO users.

In this case we should have our import chain only import eco-basic, as we don't need anything else in eco other that is-a / labels /syns

**Comment by cmungall** [comment]
All relations should have labels. Let's just filter out if we don't have a relation label for some reason. We should also show source of axioms.

In this case the relationship is simultaneously correct (if kind of weird) and of zero use to almost all AmiGO users.

In this case we should have our import chain only import eco-basic, as we don't need anything else in eco other that is-a / labels /syns

**Comment by cmungall** [comment]
Let me summarize, there are two things happening here:

 1. the display is broken, it should say "is about" not a broken glyph
 2. the relationship "cross-streak evidence is-about some response to biotic stimulus" is valid but not something we want to see in amigo.

For 1, the fix is a basic UI one. We should fall through to the label if we don't have the glyph. If there is no label don't show it. 

For 2, we have 3 options:

 * 2.1 request IAO remove this
 * 2.2 only show the same conservative set of relations as in the inferred tree view (#215)
 * 2.3 only load the basic form of IAO

I think for 2.1 IAO would claim this is valid. For 2.2, we previously decided (#220) that we would be more inclusive in the neighbor view (in contrast to tree views, where we should ONLY shows relations that make sense to view in a tree, #215). We can revisit this decision, up to the product owner @ukemi 

So that leaves 2.3 which is easy, make a ticket in go-ontology and Jim, Eric or I can take care of it.

**Comment by ukemi** [comment]
Maybe the best thing is to filter out any non-GO terms from the subject report. It seems the core of the issue here is that @ValWood and @pgaudet were confused by the presence of non-GO terms.

**Comment by ValWood** [comment]
I just think our users will be confused by what it means in this context...
